src/toil/wdl/wdltoil.py

```diff
@@ -5500,6 +5501,29 @@ def main() -> None:
                         )
                         options.all_call_outputs = True
 
+                WDL.Lint.lint(document)
+
+                strict = options.strict
```

Comment:
We probably don't need to just copy the value out of options. We could pass `options.strict` through instead.